tests/test_plugin_slack.py

```diff
@@ -74,8 +124,8 @@ def find_bot_id_query():
 
```

Comment:
The Travis `black` failure is coming from line 117, which isn't even changed in my commit:
```
-                "updated": 1502138686,
+                "updated": 1_502_138_686,
```

Reply:
I don't recall black formatting number that way. It's probably due to a newer version.